cypress/support/step_definitions/fireboltCalls.js

```diff
@@ -527,15 +527,23 @@ Given(
 );
 
 /**
- * @function User triggers event with value '{}}'
- * @description sending message to platform to make post call to set values.
- * @param {String} key - Name of event to be called.
+ * @function User triggers event with value as '(.+)'
+ * @description sending message to platform to make post call to set event values.
+ * @param {String} key - key name of the event data
  * @example
- * And User triggers event with value as ' DEVICE_ONHDCPCHANGED_EVENTS'
+ * And User triggers event with value as 'onColorimetryChanged event with colorimetry as BT2020cYCC'
  */
 Given(/User triggers event with value as '(.+)'/, (key) => {
-  fireLog.info(CONSTANTS.STEP_DEFINITION_NEEDS_TO_IMPLEMENT).then(() => {
-    throw new Error(CONSTANTS.STEP_IMPLEMENTATION_MISSING);
+  const requestMap = {
```

Comment:
any reason why we not added the logic to fetching the key value here?

Reply:
Addressed